vm/interface.go

```diff
@@ -68,7 +68,8 @@ type ValidatorSettingsHandler interface {
 	MinStepValue() *big.Int
 	TotalSupply() *big.Int
 	NumNodes() uint32
-	AuctionEnabled() bool
+	AuctionEnableNonce() uint64
+	StakeEnableNonce() uint64
 	IsInterfaceNil() bool
 }
```

Comment:
auction.go and staking.go have only ~63% code coverage. I would like a percent close to 100 for these files. Should add a new task to increase the code coverage

Reply:
agreed. will not merge to dev till close to 100% is not reached.